ddtrace/utils/formats.py

```diff
@@ -1,9 +1,12 @@
 import logging
 import os
+import re
 
 from .deprecation import deprecation
 
 
+_TAGS_NOT_SEPARATED = re.compile(r":[^,\s]+:")
```

Comment:
Worth having a comment for this IMO for those who can't read regex (me)

Reply:
You might want to provide an example or two of what we are trying to test for.

Reply:
This is documented already in the `parse_tags_str` whereas this comment is meant to be a gloss on the regex to make it easy to read.